interop-testing/src/main/java/io/grpc/testing/integration/AbstractInteropTest.java

```diff
@@ -1330,6 +1333,67 @@ public void perRpcCreds(String jsonKey, InputStream credentialsStream, String oa
     oauth2AuthToken(jsonKey, credentialsStream, oauthScope);
   }
 
+  /** Sends a cacheable unary rpc using GET. Requires that the server is behind a caching proxy. */
+  public void cacheableUnary() {
+    // Build method descriptor manually to set idempotent and safe to true.
+    io.grpc.MethodDescriptor<
```

Comment:
Why all this stuff? Why isn't CacheableUnaryCall in the proto? If it were, then I'd expect `TestServiceGrpc.METHOD_CACHEABLE_UNARY_CALL.toBuilder().setSafe(true).build()` here.

If we're going to have all this goo here, it seems at least we should remove all the unnecessary fully-qualified names (like io.grpc before MethodDescriptor).